internal/input/transactions.go

```diff
@@ -33,17 +33,18 @@ func GetTransactions(start, end uint32, limit int64, env utils.EnvironmentDetail
 	err = backend.PrepareRange(ctx, ledgerbackend.BoundedRange(start, end))
 	panicIf(err)
 	for seq := start; seq <= end; seq++ {
-		txReader, err := ingest.NewLedgerTransactionReader(ctx, backend, env.NetworkPassphrase, seq)
```

Comment:
Hmm actually the returned LedgerTransactionReader (the txReader in this case) has ledgerCloseMeta inside it. So the refactor should be to 
* remove the extra ledgerCloseMeta in line 43
* update LedgerCloseMeta in line 57 to be set by `txReader.ledgerCloseMeta`

Reply:
seem like txReader.ledgerCloseMeta field is not exported?

 https://github.com/stellar/go/blob/master/ingest/ledger_transaction_reader.go#L17

Reply:
I believe it should be
`NewLedgerTransactionReader()` calls `NewLedgerTransactionReaderFromLedgerCloseMeta()` which then sets  `ledgerCloseMeta` [here](https://github.com/stellar/go/blob/master/ingest/ledger_transaction_reader.go#L36)

Reply:
oh sorry, I might have misunderstood it. 
what I am trying to say is the ledgerCloseMeta field is not exported as it is not in capital letters. 


<img width="1266" alt="image" src="https://github.com/stellar/stellar-etl/assets/25578820/30fe1964-b23f-4981-a3a9-49db8439df65">

Reply:
Ah I missed that. You're right then